test/NuGet.CommandLine.Test/Util.cs

```diff
@@ -486,5 +504,71 @@ public static JObject CreateSinglePackageRegistrationBlob(MockServer server, str
 
             return regBlob;
         }
+
+        public static string CreateProjFileContent(
+            string projectName = "proj1",
+            string targetFrameworkVersion = "v4.5",
+            string[] references = null,
+            string[] contentFiles = null)
+        {
+            var referencesFormat = "<Reference Include='{0}' />";
+
+            var contentFileFormat = @"<Content Include = '{0}' />";
+
+            var projFileFormat = @"<Project ToolsVersion='4.0' DefaultTargets='Build'
```

Comment:
don't use raw string concat for building XML - even in test code.
XML object models are easier to write, read and update (and less fragile)
(make sure to use System.Xml.Linq)